### PR TITLE
Why did the developer have to quit smoking?

### PR DESCRIPTION
**Pull request by taylorkearns**

### CODE REVIEW

**Comment by wesbos** [comment]
i dont get it

**Comment by chalemic** [comment]
It's a pun on "sin tax", which is when the government imposes taxes on items deemed unhealthy such as cigarettes and alcohol.

**Comment by stevefan1999-personal** [comment]
They still have to, or they won't pass the smoking test

**Comment by ibnuda** [comment]
unless you're volkswagen.

File: readme.md [hunk 1]
```diff
@@ -225,3 +225,9 @@ As in: OOPs I used Java!
 **A:** Because he used up all his cache
 
 ---
+
+**Q:** Why did the developer have to quit smoking?
```
Comment (styfle) [inline readme.md]:
```suggestion
**Q:** Why did the developer decide to quit smoking?
```

File: readme.md [hunk 2]
```diff
@@ -225,3 +225,9 @@ As in: OOPs I used Java!
...
+
+**A:** Because they imposed a new syntax.
```
Comment (styfle) [inline readme.md]:
```suggestion
**A:** Because they couldn't afford to pay the new syntax.
```

Comment (styfle) [reply readme.md]: Maybe even put (Sin Tax) in parenthesis so it's a little more clear to the reader

Comment (taylorkearns) [reply readme.md]: Or maybe something like "because she heard there was a new syntax", which points out the homophone w/out having to make it obvious.

**Comment by taylorkearns** [comment]
I can also just think of a better one 😄. What's a good dad joke if not obvious!

**Comment by rjriel** [comment]
I think it works geographically. Us Canadians have never heard of a “sin tax”. I think we may just be used to the government taxing everything. It was good once it was explained though :)

**Pull request merged**